pkg/kor/exporter.go

```diff
@@ -77,3 +78,10 @@ func exportMetrics(filterOptions *filters.Options, clientset kubernetes.Interfac
 		}
 	}
 }
+
+func getUnusedResources(filterOptions *filters.Options, clientset kubernetes.Interface, apiExtClient apiextensionsclientset.Interface, dynamicClient dynamic.Interface, outputFormat string, opts Opts, resourceList []string) (string, error) {
+	if len(resourceList) > 0 {
```

Comment:
edge case here might be when a user enters all

Reply:
good catch!